bin/prettier.js

```diff
@@ -228,6 +228,16 @@ if (stdin) {
       return;
     }
 
+    if (argv["list-different"]) {
+      try {
```

Comment:
Hmm, did you forget to push your commit? This seems to be the last one on master. If you remove that `try`, I'll merge it!